config-templates/ibm/ess/2.1.0/custom-parameters.json

```diff
@@ -0,0 +1,152 @@
+[
+	{
+		"description": "The mount path of the primary file system.",
+		"displayname": "Scale Host Path",
+		"name": "scale-host-path",
+		"required": "true"
+	},
+	{
+		"description": "The cluster ID of the pirmary IBM Spectrum Scale cluster.",
+		"displayname": "Cluster ID",
+		"name": "cluster-id",
+		"required": "true",
+		"regex": "^[0-9]+$"
+	},
+	{
+		"description": "The primary file system name.",
+		"displayname": "Primary FS",
+		"name": "primary-fs",
+		"required": "true"
+	},
+	{
+		"description": "The FQDN or IP address of the GUI node on the Spectrum Scale cluster.",
+		"displayname": "GUI Host",
+		"name": "gui-host",
+		"required": "true"
+	},
+	{
+		"description": "The name of secret for that contains the username and password to connect to the primary cluster GUI.",
+		"displayname": "Primary Cluster, Secret Name",
+		"name": "secret-name",
+		"required": "true"
+	},
+	{
+		"description": "The username to connect to the primary cluster GUI.",
+		"displayname": "Primary Cluster GUI Username",
+		"name": "gui-api-user",
+		"required": "true",
+		"category": "secret"
+	},
+	{
+		"description": "The password to connect to the primary cluster GUI.",
+		"displayname": "Primary Cluster GUI Password",
+		"name": "gui-api-password",
+		"required": "true",
+		"obfuscate": "true",
+		"category": "secret"
+	},
+	{
+		"description": "The IP address of Kubernetes Node#1 running IBM Spectrum Scale.",
+		"displayname": "node1-k8-ip",
+		"name": "k8-n1-ip",
+		"required": "true",
+		"regex": "^[0-9]+[.][0-9]+[.][0-9]+[.][0-9]+$"
```

Comment:
Can you add a place-holder attribute also ?

e.g place-holder=192.168.1.1

This is a hint to user on input value format . We suggest to have this attribure for all the custom parameters that has `regex`